styles/variables-base.less

```diff
@@ -239,54 +248,54 @@
 @agate-panels-panel-padding: inherit;
 @agate-panels-panel-header-padding: @agate-panels-panel-padding;
 @agate-panels-panel-body-padding: @agate-panels-panel-padding;
-@agate-panels-tab-bar-margin: inherit;
-@agate-panels-tab-bg-position: inherit;
-@agate-panels-tab-bg-transform: inherit;
-@agate-panels-tab-border-radius: inherit;
-@agate-panels-tab-group-border-radius: inherit;
-@agate-panels-tab-group-horizontal-transform: inherit;
-@agate-panels-tab-group-margin: inherit;
-@agate-panels-tab-group-padding: inherit;
-@agate-panels-tab-group-position-after-margin: inherit;
-@agate-panels-tab-group-transform: inherit;
+@agate-panels-tab-bar-margin: 0;
+@agate-panels-tab-bg-position: 0;
+@agate-panels-tab-bg-transform: none;
+@agate-panels-tab-border-radius: 0;
+@agate-panels-tab-group-border-radius: 0;
+@agate-panels-tab-group-horizontal-transform: none;
+@agate-panels-tab-group-margin: 0;
+@agate-panels-tab-group-padding: 0;
+@agate-panels-tab-group-position-after-margin: 0;
+@agate-panels-tab-group-transform: none;
 @agate-panels-tab-margin: inherit;
 @agate-panels-tab-padding: inherit;
-@agate-panels-tab-separator-border-width: inherit;
-@agate-panels-vertical-tab-after-padding-right: inherit;
-@agate-panels-vertical-tab-before-right: inherit;
-@agate-panels-vertical-tab-bg-position: inherit;
-@agate-panels-vertical-tab-group-margin: inherit;
-@agate-panels-vertical-tab-group-padding: inherit;
-@agate-panels-vertical-tab-margin: inherit;
+@agate-panels-tab-separator-border-width: 1px;
```

Comment:
This one seems like a skin-specific candidate. Let's set this to `0` here and `1px` in the skins where it's not zero.